File: 프로그래머스/lv1/12954. x만큼 간격이 있는 n개의 숫자/x만큼 간격이 있는 n개의 숫자.cpp
#include <string>
#include <vector>

using namespace std;

vector<long long> solution(int x, int n) {
    vector<long long> answer;
    
    int count = 1; long long X = x;
    while(count <= n)
    {
        answer.push_back(X);
        ++count; if(count > n) break;
        X += x;
    }
    return answer;
}

File: 프로그래머스/1/12954. x만큼 간격이 있는 n개의 숫자/x만큼 간격이 있는 n개의 숫자.java
class Solution {
    public long[] solution(int x, int n) {
        long[] answer = new long[n];
        
        long inputVal = x;
        for(int i = 0; i < n; i++) {
            answer[i] = inputVal;
            inputVal += x;
        }
        
        return answer;
    }
}